react/hooks/useShippingOptions.ts

```diff
@@ -126,6 +133,24 @@ const useShippingOptions = () => {
     setIsPageLoading(false)
   }
 
+  const validateCartItems = async () => {
+    const orderFormId = getOrderFormId()
+
+    const products = await getCartProducts(orderFormId)
+
+    // IMPORTANT: validate products here
```

Comment:
Is this a task for the future? If so I would create a ticket an attach it to the comment so we keep track of it...

Reply:
Yes, this is! Great idea!